NYPLCardCreator/Helpers/CLPlacemark+LocationString.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  CLPlacemark+LocationString.swift
+//  Created by Ernest Fan on 2022-01-24.
+//  Copyright © 2022 NYPL. All rights reserved.
+//
+
+import Foundation
+import CoreLocation
+
+extension CLPlacemark {
+  /// This method determines if the location is within NYC, NYS, US or none of the above,
+  /// for patron creation purpose.
+  /// - Returns: A location string, eg. nyc, nys, us or an empty string
+  func getLocationString() -> String {
+    guard isoCountryCode == "US" else {
+      return ""
+    }
+    
+    guard administrativeArea == "NY" else {
```

Comment:
according to the docs this could also be spelled out, like "New York" ? also wondering if we should `lowercased()` the `administrativeArea` for safety